Dockerfile

```diff
@@ -1,7 +1,7 @@
 ARG DOCKER_HUB="docker.io"
 ARG NGINX_VERSION="1.17.6"
 
-FROM $DOCKER_HUB/library/node:10.10-alpine as build
+FROM $DOCKER_HUB/library/node:12.0-alpine as build
```

Comment:
@fllaca can you add a new ARG  NODE_VERSION and use it, as we do for Nginx, pls?

Reply:
done!